code/drasil-lang/lib/Language/Drasil/Expr/Class.hs

```diff
@@ -205,6 +210,7 @@ class ExprC r where
   -- | Create a matrix.
   matrix :: [[r]] -> r
 
+  sSet :: [r] -> r
```

Comment:
Here too: what is an `sSet`?

Reply:
It was just the name that I chose initially to avoid any name conflicts, I changed it to `set'`, it creates a set.